test2code/src/main/kotlin/com/epam/drill/test2code/Test2Code.kt

```diff
@@ -149,7 +149,7 @@ class Test2Code(
             .map {
                 ExecClassData(
                     id = it.id,
-                    className = it.name,
+                    nameId = it.nameId,
```

Comment:
Main purpose of this task is to remove the `className`, since class is already identified by `id`.
I don't get why would we want to replace it with a hash value.
Lets discuss it again, I guess.